### PR TITLE
Add IP property for event

### PR DESCRIPTION
**Pull request by shaliko**
User IP address very useful property for segment users by Country, City, Region.

amplitude_analytics

### CODE REVIEW

**Comment by shaliko** [comment]
@toothrot any feedback?

**Comment by toothrot** [comment]
@shaliko Hey! I have time after work tonight to look at this. I'll work on reviewing, testing, and getting it merged.

File: lib/amplitude_api/event.rb [hunk 1]
```diff
@@ -50,6 +55,7 @@ def to_hash
       serialized_event[:event_properties] = event_properties
       serialized_event[:user_properties] = user_properties
       serialized_event[:time] = formatted_time if time
+      serialized_event[:ip] = ip if ip
```
Comment (toothrot) [inline lib/amplitude_api/event.rb]: Won't `ip` always be at least an empty string here? Either this should check for that, or we should default to nil in `initialize`

File: lib/amplitude_api.rb [hunk 2]
```diff
@@ -28,12 +28,12 @@ class << self
     # and contains user properties to be associated with the user
     #
     # @return [ Typhoeus::Response ]
-    def send_event(event_name, user, event_properties: {}, user_properties: {})
+    def send_event(event_name, user, options = {})
```
Comment (toothrot) [inline lib/amplitude_api.rb]: Nice. I dig the consistency with Event's initialize here.

**Comment by toothrot** [comment]
Thanks!

**Pull request merged**